### DISCUSSION TITLE
About page links to 86Box

### DISCUSSION THREAD

**Discussion by Torinde** [PCBox/PCBox#91] [General]
![image](https://github.com/PCBox/PCBox/assets/97228894/10a10027-e0a0-4852-8dfb-706d47717411)
- PCBox link goes to https://86box.net/ - instead of https://pcbox-emu.xyz
- Edit in Github link is broken

**Comment by bachimusprime** [comment]
I had to break the link to Edit on GitHub because it turns out that the 86Box docs, which these docs are based on, are all rights reserved by default due to there being no LICENSE file in the repo. Haven't figured out how to take down the docs site tho.

**Reply by Torinde** [reply]
I hope I'm misunderstanding - you are not planning to delete the PCBox docs, I hope. Some parts of it cover stuff not present in 86Box docs.

Isn't it better to ask 86Box to add a LICENSE file?

**Reply by bachimusprime** [reply]
No it doesn't lol. They were literally just the 86Box docs with 86Box search-and-replaced with PCBox in many places. Also, I tried that, and they seemingly refuse to lol.

**Reply by Torinde** [reply]
For example, in [PCBox docs](https://docs.pcbox-emu.xyz/en/latest/hardware/machinespecific.html#id1) I see notes about 8088/8086 machines, which are not shown in [86Box docs](https://86box.readthedocs.io/en/latest/hardware/machinespecific.html#miscellaneous):
![image](https://github.com/PCBox/PCBox/assets/97228894/3e684ea4-29e7-4b3b-8dc5-bb3b55a788cd)
 
![image](https://github.com/PCBox/PCBox/assets/97228894/ab0c3228-4f9e-4314-a749-53e4644ab817)

**Reply by bachimusprime** [reply]
Those didn't  come from me, they came from an old version of the 86Box docs.

**Reply by Torinde** [reply]
Ah, OK, then you're right that there's no point in duplicating.

Docs for PCBox additions-related topics may be placed [here](https://github.com/PCBox/PCBox/wiki)?
